docs-source/spring/content/database/_index.md

```diff
@@ -15,12 +15,11 @@ resources:
     title: "Download ADB client credential"
 ---
 
-The Oracle Backend for Spring Boot includes an Oracle database. An instance of Oracle Autonomous Database Serverless is created during
-installation.
+The Oracle Backend for Spring Boot and Microservices includes an Oracle database. An instance of Oracle Autonomous Database Serverless is created during installation.
 
-If you chose the **Secure Access from Anywhere** option for your database during installation (or just accepted the default), then you can use
-the **Database Actions** web user interface to work with your database. If you chose the **Private** option, you need to use Bastion to access
-the database.
+> **_NOTE:_** Oracle recommends that you install your own database(s) for your production applications, the database provisioned is used for Oracle Backend for Spring Boot metadata and can be used for developement.
```

Comment:
Change "database(s)" to "databases", change the comma to a period, and init cap "The".